src/org/zaproxy/zap/view/SiteMapTreeCellRenderer.java

```diff
@@ -58,9 +62,11 @@
 	private static Logger log = Logger.getLogger(SiteMapPanel.class);
 
 	private List<SiteMapListener> listeners;
+	private JPanel component; 
 
 	public SiteMapTreeCellRenderer(List<SiteMapListener> listeners) {
 		this.listeners = listeners;
+		this.component = new JPanel();
```

Comment:
There's an extra spacing between the nodes, that might needs some tweaks (to look as before).